storey/writers.py

```diff
@@ -634,12 +634,12 @@ class WriteToTable(_ConcurrentByKeyJobExecution, _Writer):
 
     def __init__(self, table: Union[Table, str], columns: Optional[List[str]] = None, infer_columns_from_data: Optional[bool] = None,
                  **kwargs):
+        Flow.__init__(self, **kwargs)
```

Comment:
Why move the __init__ call up? The whole point of the kwargs stuff is to happen before this call.